### DISCUSSION TITLE
Running on a USB flashdrive (offline transfer)

### DISCUSSION THREAD

**Discussion by caston1981** [belowaverage-org/SuperGrate#76] [Q&A] [answered]
Can I run SuperGrate completely on a USB flashdrive to migrate from the source to target?
That is I don't use WMI over the network. I just use the flashdrive to backup the profiles from source machine and then the same drive to recover them on the target machine.

**Comment by krisdb2009** [comment, marked as answer]
Absolutely!

Drop the EXE on a flash drive and in the source computer field, click the "<" button to fill in the current PC name. Super Grate will automatically detect that this is the local PC and not use any networking to perform the migration.
When restoring a profile, select it out of the "store" and in the destination computer field, click the "<" again to fill in the current PC and then click "start".
